product/dialogs/miq_dialogs/miq_provision_redhat_dialogs_template.yaml

```diff
@@ -483,6 +483,14 @@
           :display: :edit
           :data_type: :integer
           :validation_method: :validate_memory_reservation
+        :memory_limit:
+          :description: Memory (MB)
+          :required: false
+          :notes: (Max limit is determined by Operating System type and Arch)
```

Comment:
Overall this looks good, but wondering why abbreviate "Arch" here?  If this is space related I would abbreviate "OS" (more commonly used) and spelling out "Architecture".  Otherwise don't abbreviate either.